src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -244,7 +244,7 @@ export function getRequestedAppointmentListInfo(state) {
 export function getUpcomingAppointmentListInfo(state) {
   return {
     facilityData: state.appointments.facilityData,
-    futureStatus: selectFutureStatus(state),
+    futureStatus: state.appointments.confirmedStatus,
```

Comment:
This was previously checking both confirmed and pending statuses, and we just care about the confirmed status for this page.